src/rgw/rgw_sal_motr.cc

```diff
@@ -1681,17 +1681,23 @@ MotrObject::~MotrObject() {
 //    return read_op.prepare(dpp);
 //  }
 
-int MotrObject::fetch_obj_entry_and_key(const DoutPrefixProvider* dpp, rgw_bucket_dir_entry& ent, std::string& bname, std::string& key, rgw_obj* target_obj)
+int MotrObject::fetch_obj_entry_and_key(const DoutPrefixProvider* dpp, rgw_bucket_dir_entry& ent, std::string& bname, std::string& key, rgw_obj* target_obj, RGWObjectCtx* rctx)
```

Comment:
Can you put function arguments in multiple lines line in [MotrObject::copy_object_same_zone()](https://github.com/Seagate/cortx-rgw/blob/main/src/rgw/rgw_sal_motr.cc#L2383) , its growing and doesnt look proper